audit/audit-gcp.sh

```diff
@@ -98,6 +98,29 @@ gcloud \
     | sed 's/.googleapis.com//' \
     | while read -r SVC; do
         case "${SVC}" in
+            bigquery)
+                mkdir -p "projects/${PROJECT}/services/${SVC}"
+                rm -f projects/${PROJECT}/services/${SVC}/bigquery.datasets.* || :
+                bq \
+                    --format=json --project_id=$PROJECT ls
+                    > "projects/${PROJECT}/services/${SVC}/bigquery.datasets.json"
+                
+                # Only run if there are any datasets
+                if [ -s "projects/${PROJECT}/services/${SVC}/bigquery.datasets.json" ]
+                then
+                    for DATASET in `bq --project_id=$PROJECT --format=json ls  | jq -r '.[] | .datasetReference["datasetId"]'`
```

Comment:
I'm not an expert in Bash, but few years ago I read using the backticks is being discouraged, and I think this stackoverflow has all the arguments and links about why which would be muhc better than me trying to write some reasons: https://unix.stackexchange.com/a/126928
but maybe there is a reason why you used them in the first place, so it's just a comment and maybe invitation to discussion about it. Wdyt?

Reply:
Nice observation Bart. I had not heard of this before but I will of course take that into account. I'll refactor that code to get rid of them.

Reply:
Thank you! :-)